api/turing/api/router_versions_api.go

```diff
@@ -177,6 +180,24 @@ func (c RouterVersionsController) DeployRouterVersion(
 		"version":   int(routerVersion.Version),
 	})
 }
+func (c RouterVersionsController) ListEnsemblerRouterVersion(
```

Comment:
Hmm this name doesn't sound too descriptive about what it does, can we rename it to something like `ListRouterVersionsByEnsembler` or `ListRouterVersionsWithEnsembler`?